sql/core/src/main/scala/org/apache/spark/sql/RuntimeConfig.scala

```diff
@@ -36,6 +37,7 @@ class RuntimeConfig private[sql](sqlConf: SQLConf = new SQLConf) {
    * @since 2.0.0
    */
   def set(key: String, value: String): Unit = {
+    assertNotGlobalSQLConf(key)
```

Comment:
Sorry, I did not explain it clearly. What I meant above is to add `assertNotGlobalSQLConf` to  [the other two set functions](https://github.com/cloud-fan/spark/blob/59fc4e996c60359e3d1b232fa3699c98a3f27f9e/sql/core/src/main/scala/org/apache/spark/sql/RuntimeConfig.scala#L44-L60) Does this make sense?